AppCalc/src/main/java/ua/goit/timonov/enterprise/AppConfig.java

```diff
@@ -9,52 +9,59 @@
  * Java-based Spring configuration class for AppCalc
  */
 
-@Configuration
-@EnableAspectJAutoProxy
-@Import(LogAspect.class)
+//@Configuration
```

Comment:
OK, I've deleted this class with Spring config - whole config is in application-context.XML